src/main/java/com/jagrosh/vortex/commands/moderation/SlowmodeCmd.java

```diff
@@ -0,0 +1,126 @@
+import java.util.concurrent.TimeUnit;
+
+/**
+ * @author Michaili K (mysteriouscursor+git@protonmail.com)
+ */
+public class SlowmodeCmd extends ModCommand
+{
+    private final static int MAX_SLOWMODE = 21600;
+
+    public SlowmodeCmd(Vortex vortex)
+    {
+        super(vortex, Permission.MANAGE_CHANNEL);
+        this.name = "slowmode";
+        this.arguments = "[time or OFF] | [time to disable slowmode]";
+        this.help = "enables or disables slowmode";
+        this.botPermissions = new Permission[]{Permission.MANAGE_CHANNEL};
+        this.guildOnly = true;
+    }
+
+    @Override
+    protected void execute(CommandEvent event)
+    {
+
+        if(event.getArgs().isEmpty())
+        {
+            int slowmodeDuration = vortex.getDatabase().tempslowmodes.timeUntilDisableSlowmode(event.getTextChannel());
+            int slowmodeTime = event.getTextChannel().getSlowmode();
+
+            if(slowmodeTime <= 0)
+            {
+                event.reply("Slowmode is disabled.");
+                return;
+            }
+
+            if(slowmodeDuration <= 0)
+                event.reply("Slowmode is enabled with 1 message every "+FormatUtil.secondsToTimeCompact(slowmodeTime)+".");
+            else
+                event.reply("Slowmode is enabled with 1 message every "+FormatUtil.secondsToTimeCompact(slowmodeTime) +
+                        " for "+FormatUtil.secondsToTimeCompact(slowmodeDuration)+".");
+            return;
+        }
+
+        String args = event.getArgs();
+
+        if(args.equals("0") || args.equalsIgnoreCase("off"))
+        {
+            vortex.getDatabase().tempslowmodes.clearSlowmode(event.getTextChannel());
+            event.getTextChannel().getManager().setSlowmode(0).queue();
+            event.replySuccess("Disabled slowmode!");
+            return;
+        }
+
+        String[] split = args.split("\\|",2);
+
+        int slowmodeTime = OtherUtil.parseTime(split[0]);
+        if(slowmodeTime == -1)
+        {
+            event.replyError("Invalid slowmode time!");
+            return;
+        }
+        if(slowmodeTime > MAX_SLOWMODE)
+        {
+            event.replyError("You can only enable slowmode for up to 6 hours!");
+            return;
+        }
+        if(slowmodeTime < -1)
+        {
+            event.replyError("Slowmode cannot use negative time!");
+            return;
+        }
+
+        int slowmodeDuration = split.length == 1 ? 0 : OtherUtil.parseTime(split[1]);
+        if(slowmodeDuration == -1)
+        {
+            event.replyError("Invalid slowmode duration time!");
+            return;
+        }
+        if(slowmodeDuration < -1)
+        {
+            event.replyError("Slowmode duration cannot use negative time!");
+            return;
+        }
+
+        event.getTextChannel().getManager()
+                .setSlowmode(slowmodeTime)
+                .reason("Enabled by "+event.getAuthor().getAsTag()+" ("+event.getAuthor().getId()+")")
```

Comment:
In case we want to put this in modlog later, this should use `LogUtil.auditReasonFormat` for the reason. If we do add that, we'll also want to support including a reason in the command syntax, but I'm not worried about that right now.
```suggestion
                .reason(LogUtil.auditReasonFormat(event.getMember(), slowmodeDuration/60, "Enabled slowmode"))
```